### PR TITLE
Small updates to home form

### PR DESCRIPTION
**Pull request by loganmorganml1**
made some general cleanup to the homepage form. 
-- centered the date placeholder text 
-- brought up the content so it shows better on the opening page
--cleaned up the about button to be left aligned for tablet and desktop 
--added bold roboto font weight to all pages

### CODE REVIEW

**Review by lala-laura — APPROVED** [review]
Looks good

File: assets/css/style.css [hunk 1]
```diff
@@ -1701,6 +1701,7 @@ margin: -20px 0px 50px 0px;
 }
 .search-form-item{
   padding-bottom: 20px;
+  font-family: ;
```
Comment (lala-laura) [inline assets/css/style.css]: Only thing I see is the missing font-family

Comment (loganmorganml1) [reply assets/css/style.css]: The new font weight is in the HTML from the google link

Comment (loganmorganml1) [reply assets/css/style.css]: Sorry this must have been an error- just deleted it!

**Pull request merged**